app/src/main/java/com/framgia/forder/screen/shopmanagement/ShopManagementFragment.java

```diff
@@ -8,7 +8,19 @@
 import android.view.View;
 import android.view.ViewGroup;
 import com.framgia.forder.R;
+import com.framgia.forder.data.model.Shop;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.model.Shop. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports